src/test/ensemble_flow_custom_node_tests.cpp

```diff
@@ -2106,6 +2107,231 @@ TEST_F(EnsembleConfigurationValidationWithDemultiplexer, SuccessfulConfiguration
     ASSERT_EQ(pipelineDefinition->validate(manager), StatusCode::OK);
 }
 
+TEST_F(EnsembleConfigurationValidationWithDemultiplexer, SuccessfulConfigurationSingleDynamicDemultiplexerFirst) {
+    std::optional<uint32_t> demultiplyCount = 0;
+
+    std::vector<NodeInfo> info{
+        {NodeKind::ENTRY, ENTRY_NODE_NAME, "", std::nullopt, {{pipelineInputName, pipelineInputName}}},
+        {NodeKind::CUSTOM, "custom_node_1", "", std::nullopt, {{"1", "out_OutputNumbers_1"}, {"2", "out_OutputNumbers_2"}}, demultiplyCount, {}, mockedLibrary, parameters_t{{"in_InputNumbers", "1,10;FP32"}, {"out_OutputNumbers_1", "1,0,10;FP32"}}},
+        {NodeKind::CUSTOM, "custom_node_2", "", std::nullopt, {{"out", "out_OutputNumbers"}}, std::nullopt, {}, mockedLibrary, parameters_t{{"in_InputNumbers_1", "1,10;FP32"}, {"out_OutputNumbers", "1,10;FP32"}}},
```

Comment:
It would be clearer to read to separate each metadata in new line like in tests above, for example L2088